src/Presentation/FacetUiBuilder.php

```diff
@@ -148,31 +145,29 @@ private function getExampleRangeItems(): array {
 	/**
 	 * @param array<array<string, mixed>> $items
 	 */
-	private function getItemsHtml( array $items, string $type, string $propertyId ): string {
```

Comment:
@alistair3149 we have the Enum so we can use it as a type. We only transition to the string value when calling code we do not control or where the PHP type is not possible, such as when handing over to the template engine.